### PR TITLE
Add doc for BF.DEBUG

### PR DESCRIPTION
**Pull request by bugwz**

### CODE REVIEW

File: docs/commands/bf.debug.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+* **bits**: The bits size of every filter.
+* **hashes**: The number of hash functions.
+* **hashwidth**: 64-bit hash or 32-bit hash.
+* **capacity**: The entries size of every filter.
+* **size**: The number of items in the filter.
+* **ratio**: The error ratio of filter.
+
+@examples
+
+```
```
Comment (ashtul) [inline docs/commands/bf.debug.md:20]: Thank you for the contribution.
Please remove extra new lines

**Review by ashtul — COMMENTED** [review]
Thank you.
Small style comment.

File: docs/commands/bf.debug.md (hunk 1 above)
Comment (bugwz) [reply docs/commands/bf.debug.md]: Thanks~ done~

**Comment by LiorKogan** [comment]
Let us leave BF.DEBUG undocumented for now.
We should have the same commands for all data structures.
+ DEBUG is not a good name here.

**Pull request closed without merging**